testgrid/cmd/summarizer/BUILD.bazel

```diff
@@ -1,8 +1,11 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 go_library(
   name = "go_default_library",
-  srcs = ["summary.go"],
+  srcs = [
+      "summarizer.go",
+      "summary.go",
```

Comment:
Consider renaming `summary.go` to `types.go`

A pattern I have noticed in this repo and kubernetes/kubernetes is to drop all of the structs and types in a file named `types.go`

Reply:
Make sense, thanks. The goal is also to move this out later when the structs need to be shared, so it does make sense to call it something more general.